WalletWasabi.Fluent/ViewModels/Wallets/Send/PrivacySuggestionsFlyoutViewModel.cs

```diff
@@ -33,7 +35,7 @@ public PrivacySuggestionsFlyoutViewModel()
 
 	public ObservableCollection<SuggestionViewModel> Suggestions { get; }
 
-	public  async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo info, BitcoinAddress destination, BuildTransactionResult transaction)
+	public async Task BuildPrivacySuggestionsAsync(Wallet wallet, TransactionInfo info, BitcoinAddress destination, BuildTransactionResult transaction, CancellationToken cancellationToken = default)
```

Comment:
This always has to have a cancellationToken IMO. Remove the default to make sure it is always passed because this can take ages in some cases. For example, it is sure that if the user cancels the send workflow this needs to be stopped - currently, cancellation is not happening in some cases.